installing/installing_openstack/installing-openstack-network-config.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="installing-openstack-network-config"]
+= Configuring network settings after installing OpenStack
```

Comment:
Not sure why you singled-out OpenStack for this new assembly.  Do you plan to do more install platforms later?

Reply:
It's because the content that is being moved is specific to OpenStack. I have discussed it with @maxwelldb who is the writer for that section, and it was agreed that it could move to the OpenStack section rather than staying in the general postinstallation section, since it only applies for this platform.
This content is not new, it's just being moved from one section to a new assembly 🙂